python/lsst/ts/externalscripts/base_make_calibrations.py

```diff
@@ -1036,45 +1159,59 @@ async def arun(self, checkpoint=False):
                             verify_tests_pass = report_check_verify_stats[
                                 "CERTIFY_CALIB"
                             ]
-                            verify_report = report_check_verify_stats["NUM_STAT_ERRORS"]
-                            thresholds_report = report_check_verify_stats[
-                                "FAILURE_THRESHOLDS"
-                            ]
-                            verify_stats = report_check_verify_stats["VERIFY_STATS"]
-
+                            gen_collection = f"u/ocps/{job_id_calib}"
+                            verify_collection = f"u/ocps/{job_id_verify}"
                             if verify_tests_pass:
                                 self.log.info(
                                     f"{im_type} calibration passed verification criteria "
                                     "and will be certified."
                                 )
+                                self.log.info(
+                                    f"Generation collection: {gen_collection}"
+                                )
+                                self.log.info(
+                                    f"Verification collection: {verify_collection}"
+                                )
+
                                 await self.certify_calib(im_type, job_id_calib)
-                            elif verify_tests_pass and verify_report is not None:
+                            elif (
+                                verify_tests_pass
+                                and report_check_verify_stats["NUM_STAT_ERRORS"]
+                                is not None
+                            ):
                                 await self.certify_calib(im_type, job_id_calib)
                                 self.log.warning(
                                     f"{im_type} calibration passed the overall verification "
-                                    " criteria and was certified, but the following tests did not pass: "
-                                    f"{verify_report}\n"
-                                    f"{verify_stats}"
+                                    " criteria and was certified, but the are tests that did not pass: "
+                                )
+                                final_report_string = (
+                                    await self.build_verification_report_summary(
+                                        report_check_verify_stats
+                                    )
+                                )
+                                self.log.error(final_report_string)
+                                self.log.error(
+                                    f"Generation collection: {gen_collection}"
+                                )
+                                self.log.error(
+                                    f"Verification collection: {verify_collection}"
```

Comment:
Are you sure you want these to be 3 different log messages? I'm a bit uneasy about that because they apparently relate to each other, but once they are published as 3 different messages that becomes harder to reconstruct later.

Reply:
I'll unify the messages